README.md

```diff
@@ -0,0 +1,150 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  Fruit Tree
+</div>
+
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features) 
+    - [🚀 Live Demo](#live-demo)
+    - [🚀 video presentation](#video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Fruit Tree] <a name="Bookstore"></a>
+
+
+
+**[Fruit Tree]** A React Redux application.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<ul>
+    <li>Linters</li>
+    <li>React</li>
+    <li>Node.js</li>
+    <li>Jest</li>
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Linter implementation**
+- **React implementation**
+- **Test implementation using Jest snapshots and React Testing Library**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live demo](https://65029051474b2c132566a13d--velvety-hamster-d0b4f7.netlify.app/)
+
+
+ ## 🚀 Video Presentation <a name="video"></a>
+
+- [Video](https://www.loom.com/share/b8b4dace106a40d18b1642a9e188af32?sid=08d83a11-8e9a-42fe-8495-506219f15e55)
+
+ ## 🔭 Future Features <a name="future-features"></a>
+
+- **[I will try to add some more user interactivity using react]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="start-project"></a>
+
+### Prerequisites
+
+In order to run this project you need: 
+- Node js
+- git
+
+Installed in your systeme.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/AjroudiRida/fruittree.git
+
+### Install
+
+Install this project with:
+
+npm install
+
+### Usage
+
+To run the project, execute the following command: 
+
+
+npm start
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+npm test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="Ahmed"></a>
+
+👤 **Rida**
+
+- GitHub: [@githubhandle](https://github.com/AjroudiRida)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Make the Fruit Tree responsive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](http://ridaajroud.me/Bookstore/issues).
```

Comment:
The link to the `issues` page is broken, kindly fix that. Let's avoid broken links in our readme for a professional repository. 😊

![image](https://github.com/AjroudiRida/fruittree/assets/78251927/4a3ca2d3-ccf5-4782-a1ed-8d05df263076)